src/components/structures/InteractiveAuth.js

```diff
@@ -140,19 +140,21 @@ export default React.createClass({
         });
     },
 
-    _requestCallback: function(auth) {
+    _requestCallback: function(auth, background) {
         this.setState({
-            busy: true,
+            busy: !background,
```

Comment:
sorry, I'm not quite grokking this. why do we only set `busy` if not `background`?